packages/utils/lib/config-manager.js

```diff
@@ -133,58 +114,74 @@ export class ConfigManager {
 	isLoaded() {
 		return this._isLoaded;
 	}
+
+	/**
+	 * @param {string} filePath
+	 * @returns {string | null} The absolute path to the file or null if it doesn't exist.
+	 * @private
+	 */
+	static _fileExistsRecursive(filePath) {
```

Comment:
Minor comment: Given this name I would expect this to return a `boolean`. How about something like `_findFirstFileRecursive()`?

Reply:
Good call! Updated